src/celery_prometheus/prometheus_bootstep.py

```diff
@@ -1,37 +1,35 @@
 """Helper for celery."""
 import logging
+from argparse import ArgumentParser
+from typing import Any, Mapping, Optional, cast
 
-from celery import VERSION as celery_version
+from celery import VERSION as celery_version  # type: ignore
 from celery import Celery
-from prometheus_client import REGISTRY, start_http_server
-from prometheus_client.multiprocess import MultiProcessCollector
+from prometheus_client import REGISTRY, start_http_server  # type: ignore
```

Comment:
Update the prometheus_client to ^0.16.0 from the dev depenencies in order to get typing.